src/components/toolbar-button.tsx

```diff
@@ -6,6 +6,7 @@ export interface IButtonProps {
   toolButton: IToolbarButtonModel;
   isActive: boolean;
   isDisabled: boolean;
+  isPrimary?: boolean;
```

Comment:
what is a "primary" button?

Reply:
In the spec the toolbar has some buttons that are highlighted in the color of the section and some that are not.  I used "primary" to distinguish them.

![image](https://github.com/user-attachments/assets/f8c53305-504c-4dad-a57b-82f97a82afaf)

Reply:
I guess the point of the question was to document it somewhere in the code.